deployment/script/build-CCNP.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -o errexit
+
+ccnp_url=https://github.com/intel/confidential-cloud-native-primitives.git
+device_plugin=ccnp-device-plugin:0.1
+get_quote=ccnp-quote-server:0.1
+measurement=ccnp_measurement_server:0.1
+eventlog=ccnp_eventlog_server:0.1
+
+add_proxy(){
+        sed -i '/FROM .* as [b,q]/I a\ENV https_proxy='"$HTTPS_PROXY"''  "$1"
+        sed -i '/FROM .* as [b,q]/I a\ENV http_proxy='"$HTTP_PROXY"''  "$1"
+        sed -i '/FROM .* as [b,q]/I a\ENV HTTPS_PROXY='"$HTTPS_PROXY"''  "$1"
+        sed -i '/FROM .* as [b,q]/I a\ENV HTTP_PROXY='"$HTTP_PROXY"''  "$1"
+}
+
+#Download and install helm
+curl https://baltocdn.com/helm/signing.asc | sudo apt-key add -
+sudo apt-get install apt-transport-https --yes
+echo "deb https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+sudo apt-get update
+sudo apt-get install helm
+
+#Download CCNP
+git clone $ccnp_url
```

Comment:
When users execute this script, they should already clone the repo, why they need to clone it again?